test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -74,7 +74,7 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
             dotnet.Exec("exec", "--runtimeconfig", runtimeConfig, appDll)
                 .CaptureStdErr()
                 .CaptureStdOut()
-                .Execute()
+                .Execute(true)
```

Comment:
i would like these calls to use named parameters, i.e
```C#
.Execute(fExpectedToFail:true)
```
so that people reading the tests can understand the intent easily